src/box/session.h

```diff
@@ -125,6 +125,7 @@ struct session {
 	struct credentials credentials;
 	/** Trigger for fiber on_stop to cleanup created on-demand session */
 	struct trigger fiber_on_stop;
+	struct rlist in_active_list;
```

Comment:
All fields in this struct have comments, if you add some field it's need comment too.

Reply:
Fix